src/util/strutil.c

```diff
@@ -0,0 +1,33 @@
+//
+// Created by alon on 07/10/2020.
+//
+#include "strutil.h"
+#include <string.h>
+#include <ctype.h>
+#include <assert.h>
+
+
+void str_tolower(const char *str, char *lower, short *lower_len) {
+    size_t str_len = strlen(str);
+    /* Avoid overflow. */
+    assert(*lower_len > str_len);
+
+    /* Update lower len*/
```

Comment:
space between closing comment `*/` and the comment itself.
`/* Update lower len */`